From 469e70178daa7bc69ce9090907d9576e128525a5 Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Thu, 1 Jun 2017 15:18:30 +0500
Subject: [PATCH] CMakeLists: only deploy headers when needed

In a scenario where Vulkan loader and layers
are deployed through their own package trying
to deploy headers again faces failures.
We now make this conditional for such scenarios.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 10c1b8150..bce30fbc8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -356,7 +356,7 @@ if(NOT WIN32)
     endif()
 endif()
 
-if(UNIX)
+if(UNIX AND BUILD_LOADER)
     install(DIRECTORY "${PROJECT_SOURCE_DIR}/include/vulkan" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
 # uninstall target
-- 
2.11.1

